deployments/base-goerli/weth/deploy.ts

```diff
@@ -0,0 +1,70 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet, exp } from '../../../src/deploy';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const deployed = await deployContracts(deploymentManager, deploySpec);
+  return deployed;
+}
+
+async function deployContracts(
+  deploymentManager: DeploymentManager,
+  deploySpec: DeploySpec
+): Promise<Deployed> {
+  const trace = deploymentManager.tracer();
+
+  // Deploy constant price feed for WETH
+  const wethConstantPriceFeed = await deploymentManager.deploy(
+    'WETH:priceFeed',
+    'pricefeeds/ConstantPriceFeed.sol',
+    [
+      8,                                             // decimals
+      exp(1, 8)                                      // constantPrice
+    ]
+  );
+
+  // Deploy scaling price feed for cbETH
+  // XXX there is no cbETH / ETH pricefeed on testnet
+  // const cbETHScalingPriceFeed = await deploymentManager.deploy(
```

Comment:
did you intend to leave this commented-out price feed in?

Reply:
Yeah I did this on purpose just so when I work on the mainnet deploy script, I'll remember that it'll be different from the testnet one.